src/tablo/plugins/columnsReorder/DNDHeader.js

```diff
@@ -90,18 +92,23 @@ const columnType = ({ tabloUniqueId }) => `${tabloUniqueId}_column`;
 })
 export default class DNDHeader extends React.Component {
 
-  template({ connectDragSource, connectDragPreview, connectDropTarget, isDragAllowed, isDragging, canDrop, isOver, children }) {
-    return connectDropTarget(connectDragPreview(
+  getLocals({ connectDragSource, connectDragPreview, connectDropTarget, isDragAllowed, ...props }) {
+    const dndWrapper = isDragAllowed ? flowRight(connectDropTarget, connectDragPreview, connectDragSource) : identity;
```

Comment:
Minor: if you don't specify any `connectDragPreview`, the default preview should be the component wrapped in `connectDragSource`, so you could omit `connectDragPreview` call.
It's up to you if remove it or not.